packages/atlas-service/src/util.ts

```diff
@@ -78,63 +78,95 @@ export async function throwIfNotOk(
 }
 
 export type AtlasServiceConfig = {
+  /**
+   * MongoDB Driver WebSocket proxy base url
+   */
+  wsBaseUrl: string;
+  /**
+   * Cloud UI backend base url
+   */
+  cloudBaseUrl: string;
+  /**
+   * Atlas admin API base url
+   */
   atlasApiBaseUrl: string;
-  atlasApiUnauthBaseUrl: string;
+  /**
+   * Atlas OIDC config
+   */
   atlasLogin: {
     clientId: string;
     issuer: string;
   };
+  /**
+   * Atlas Account Portal UI base url
+   */
   authPortalUrl: string;
 };
 
 /**
  * Atlas service backend configurations.
- *  - compass-dev: locally running compass kanopy backend (localhost)
- *  - compass:    compass kanopy backend (compass.mongodb.com)
- *  - atlas-local: local mms backend (localhost)
- *  - atlas-dev:  dev mms backend (cloud-dev.mongodb.com)
- *  - atlas:      mms backend (cloud.mongodb.com)
+ *  - atlas-local:             local mms backend         (localhost)
+ *  - atlas-dev:               dev mms backend           (cloud-dev.mongodb.com)
+ *  - atlas:                   mms backend               (cloud.mongodb.com)
+ *  - web-sandbox-atlas-local: local mms backend + proxy (localhost / proxy prefix)
+ *  - web-sandbox-atlas-dev:   dev mms backend + proxy   (cloud-dev.mongodb.com / proxy prefix)
+ *  - web-sandbox-atlas:       mms backend + proxy       (cloud.mongodb.com / proxy prefix)
  */
 const config = {
-  'compass-dev': {
-    atlasApiBaseUrl: 'http://localhost:8080',
-    atlasApiUnauthBaseUrl: 'http://localhost:8080',
+  'atlas-local': {
+    wsBaseUrl: 'ws://localhost:61001/ws',
+    cloudBaseUrl: '',
+    atlasApiBaseUrl: 'http://localhost:8080/api/private',
     atlasLogin: {
-      clientId: '0oajzdcznmE8GEyio297',
-      issuer: 'https://auth.mongodb.com/oauth2/default',
+      clientId: '0oaq1le5jlzxCuTbu357',
```

Comment:
This is an id that Okta (or any other oidc provider) would issue when you are creating a new oauth supporting application. Like if you go to the admin panel of the [auth portal](https://auth-qa.mongodb.com/app/UserHome) (I'm linking to qa one here because none of us have access to production one), you'd be able to find compass app there and it will have this id

We went back and forth on this initially, but this is sort of public stuff anyway and easier to keep hardcoded here rather than somewhere else, like evergreen env. We also double-checked with security that it's okay to keep open like that before doing this 😄